src/main/java/org/jenkinsci/plugins/durabletask/ProcessLiveness.java

```diff
@@ -74,13 +83,13 @@ public static boolean isAlive(VirtualChannel channel, int pid, Launcher launcher
     private static boolean _isAlive(VirtualChannel channel, int pid, Launcher launcher) throws IOException, InterruptedException {
         if (launcher instanceof Launcher.LocalLauncher || launcher instanceof Launcher.RemoteLauncher) {
             try {
-                boolean alive = channel.call(new Liveness(pid));
-                LOGGER.log(Level.FINER, "{0} is alive? {1}", new Object[] {pid, alive});
-                return alive;
-            } catch (RuntimeException x) {
+                return channel.call(new Liveness(pid));
```

Comment:
You could probably check Jenkins core version here: `Jenkins.getVersion().isNewerThan(new VersionNumber("2.64"))` in order to optimize the call on the agent side. and to check JNR version before it . But it's rather polishing, which may actually break some use-cases with pre-loaded JAR files

Reply:
Let's hold off on this.